core/domain/exp_domain.py

```diff
@@ -1692,7 +1697,7 @@ def create_default_exploration(
             exploration_id, title, category, objective, language_code, [], '',
             '', feconf.CURRENT_EXPLORATION_STATES_SCHEMA_VERSION,
             feconf.DEFAULT_INIT_STATE_NAME, states_dict, {}, [], 0,
-            feconf.DEFAULT_AUTO_TTS_ENABLED)
+            feconf.DEFAULT_AUTO_TTS_ENABLED, False)
```

Comment:
Worth defining constant for?

Reply:
Yeah, thought about it -- but I don't think so. I don't see us ever wanting to change it; it feels more like the "0" that's an earlier argument in the list.